PosteAPI/models.py

```diff
@@ -65,3 +98,23 @@ class Post(models.Model):
 
     def __str__(self):
         return self.title
+
+
+# why gettext_lazy?
+# https://stackoverflow.com/questions/54802616/how-can-one-use-enums-as-a-choice-field-in-a-django-model
```

Comment:
Something I've seen once or twice but haven't really used in practice. Thanks for the info